packages/react-ui-components/src/MultiSelectBox_ListPreviewSortable_DraggableListPreviewElement/multiSelectBox_ListPreviewSortable_DraggableListPreviewElement.js

```diff
@@ -129,7 +137,11 @@ export default class MultiSelectBox_ListPreviewSortable_DraggableListPreviewElem
                             hoverStyle={'clean'}
                             />
                     )}
-                    <div className={theme.selectedOptions__innerPreview}>
+                    <div
+                        className={theme.selectedOptions__innerPreview}
+                        onClick={this.handleClick}
+                        role="button"
```

Comment:
```suggestion
                        role={this.props.onItemClick ? "button" : null}
```

Likewise, `role="button"` only makes sense, if the element is actually supposed to trigger an action on click.